src/pl/senla/task9/ex3/Constant.java

```diff
@@ -0,0 +1,7 @@
+package pl.senla.task9.ex3;
+
+public class Constant {
+
+    public static final Integer MAX_BUFFER_CAPACITY = 3;
+    public static final Integer MAX_PRODUCTION      = 10;
```

Comment:
Program should run endlessly, no need to limit production and complete execution of program.
The only important thing is not to overflow the buffer. And so that the producer and consumer can wait for it

Reply:
edited